modules/claims_api/app/swagger/claims_api/v2/veterans/claims_controller_swagger.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    module Veterans
+      class ClaimsControllerSwagger
+        include Swagger::Blocks
+
+        swagger_path '/veterans/{veteran_id}/claims' do
```

Comment:
lets me consistent with our camelCase

Reply:
Ah, good call.
Changed [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6955/commits/e853ffbe0b7d9efdbebb9e96ca3ee927abe34c2b).